extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -107,7 +114,18 @@ public static PhotoModel convertToPhotoModel(
         albumId.orElse(null),
         false  /*inTempStore*/,
         null  /*sha1*/,
-        mediaItem.getUploadedTime());
+        getCreationTime(mediaItem));
+  }
+
+  private static Date getCreationTime(GoogleMediaItem mediaItem) throws ParseException  {
+    // cannot be empty or null based. Verified the backend code.
+    try {
+      return CREATION_TIME_FORMAT.parse(mediaItem.getMediaMetadata().getCreationTime());
+    } catch (ParseException parseException) {
+      throw new ParseException(String.format("Failed to parse the string %s to get creationTime. "
+              + "Let's look into the dataFormatter",
```

Comment:
nit: I'd remove "Let's look into the dateFormatter" but your choice

Reply:
the parseException.getErrorOffset isnt being inserted into the string anywhere

Reply:
Oh it another parameter to throw the exception and not part of String format